### DISCUSSION TITLE
Just Reporting: codeprojectai consistent misrecognition

### DISCUSSION THREAD

**Discussion by hspindel** [codeproject/CodeProject.AI-Server#215] [General]
I am using codeprojectai with Blue Iris.  Very consistently, dogs, cats, and raccoons are recognized as persons.  Maybe there is something I misconfigured?

Blue Iris is configured to confirm alerts with ai for person and animal 50% confidence, 1 pre-trigger image, 2 post-trigger images.

**Comment by langas6** [comment]
In CPAI, have you checked the size of the model? If it's set to "tiny" then there's probably not enough detail in the image it's processing. Change to Small as a minimum or Medium if your hardware can still inference the image at less than 100ms.

There's a tonne of options to play around with in BI to make things work properly. I spent a very long time working out these settings. If need be, setup an account and check into the Blue Iris forum, plenty of posts in there to help learn what does what.

**Reply by hspindel** [reply]
I never changed the size of the model so I don't know what it is.  I'm going to try setting it to medium now.

Thank you for the reply.

**Comment by ChrisMaunder** [comment]
This is a model-dependant issue, not a code issue. The larger the model, the better the results, and the more specific a model, the better the results. 

You haven't said which module you're using so we can't suggest a model that would be better suited.

**Reply by hspindel** [reply]
Sorry.  I'm using YOLOv5.net, module Object Detection.  I have increased the model size to medium.  Will have to wait until an animal wanders across my property to see if that fixed anything.

Thank you.
